utils/internal/download_gigaspeech_from_tsinghua.sh

```diff
@@ -10,9 +10,9 @@ with_dict=false
 
 . ./utils/parse_options.sh || exit 1
 
-if [ $# -ne 1 ]; then
-  echo "Usage: $0 <gigaspeech-dataset-dir>"
-  echo " e.g.: $0 /disk1/audio_data/gigaspeech"
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 <gigaspeech-dataset-dir><host>"
```

Comment:
Same here, you don't need `host`. You can hard-code it to `tsinghua` in the internal code.